src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js

```diff
@@ -17,50 +17,113 @@ fragment vetCenterLocationsFragment on NodeVetCenterLocationsList {
   fieldNearbyVetCenters {
     entity {
       ... on NodeVetCenter {
+        title
         entityBundle
         fieldAddress {
           locality
           administrativeArea
           postalCode
           addressLine1
+          organization
         }        
         fieldPhoneNumber
         ${derivativeImage('_32MEDIUMTHUMBNAIL')}                           
       }          
       ... on NodeVetCenterOutstation {
+        title
         entityBundle
         fieldAddress {
           locality
           administrativeArea
           postalCode
           addressLine1
+          organization
         }
         fieldPhoneNumber
         ${derivativeImage('_32MEDIUMTHUMBNAIL')}             
       }
       ... on NodeVetCenterCap {
+        title
         entityBundle
         fieldAddress {
           locality
           administrativeArea
           postalCode
           addressLine1
+          organization
         }        
         ${derivativeImage('_32MEDIUMTHUMBNAIL')}                           
       }
       ... on NodeVetCenterMobileVetCenter {
+        title
         entityBundle              
         fieldAddress {
           locality
           administrativeArea
           postalCode
           addressLine1
+          organization
         }      
         fieldPhoneNumber
         ${derivativeImage('_32MEDIUMTHUMBNAIL')}                            
       }              
     } 
   }
+  fieldNearbyMobileVetCenters {
+    entity {
+      ... on NodeVetCenter {
+        title
+        entityBundle
+        fieldAddress {
+          locality
+          administrativeArea
+          postalCode
+          addressLine1
+          organization
```

Comment:
At some point we should DRY up this code using something like the `derivativeImage` function or a fragment.

Reply:
will refactor